content/docs/beyond-the-basics/public-attestation/index.md

```diff
@@ -1,14 +1,175 @@
 ---
- title: "Public Assets vs Private Assets"
- description: "Transparency through Public Attestation"
+ title: "Public Attestation"
+ description: "Public Assets vs Private Assets"
  lead: "Transparency through Public Attestation"
  date: 2021-05-18T14:52:25+01:00
  lastmod: 2021-05-18T14:52:25+01:00
- draft: true
+ draft: false
  images: []
  menu:
    docs:
      parent: "beyond-the-basics"
- weight: 41
- toc: false
----
\ No newline at end of file
+ weight: 42
+ toc: true
+---
+
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, the vulnerability reports against an OpenSource software package or the maintenance records for a building.
```

Comment:
"For example, the vulnerability reports against an OpenSource software package or the maintenance records for a building."

Can we say these are things you may wish to attest to? 

Propose "For example, you may attest to data containing a vulnerability report against an OpenSource software package or maintanence records for a building."

Reply:
Change implemented